src/orders/Dockerfile

```diff
@@ -37,7 +37,7 @@ RUN gradle --refresh-dependencies
 FROM buildenv as appbuild
 ARG APPLICATION_NAME
 ARG VERSION
-
+ENV INSTANT_CLIENT_VERSION=${clientVersion}
```

Comment:
Where this is used?

Reply:
Did not stage that. Will Un stage this hunk as well. Was working on parameteizing instant client version

Reply:
Remember that the version can also affect the wallet volume

Reply:
Yes - the parametrization is not in this PR. Removed it.